frontend/vite.config.ts

```diff
@@ -0,0 +1,34 @@
+import { fileURLToPath } from 'node:url'
+import { dirname, resolve } from 'node:path'
+
+import { defineConfig, loadEnv } from 'vite'
+import viteReact from '@vitejs/plugin-react'
+import tailwindcss from '@tailwindcss/vite'
+import { tanStackRouterCodeSplitter } from '@tanstack/router-plugin/vite'
+
+const __dirname = dirname(fileURLToPath(import.meta.url))
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd(), 'VITE_')
+  return {
+    plugins: [
+      tanStackRouterCodeSplitter(),
```

Comment:
w docsach to jakos inaczej jest zasetupowane ale nwm jaka roznica https://tanstack.com/router/latest/docs/framework/react/routing/installation-with-vite#configuration